ru/lessons/advanced/metaprogramming.md

```diff
@@ -0,0 +1,238 @@
+---
+layout: page
+title: Метапрограммирование
+category: advanced
+order: 7
+lang: ru
+---
+
+Метапрограммирование - процесс написания кода с помощью другого кода. В Elixir это дает нам возможность расширять возможности языка для большего соответствия нуждам приложения. Мы начнем этот урок с рассмотрения внутреннего представления Elixir кода. А уже потом - узнаем как его модифицировать. И, в итоге, мы применим эти знания для создания собственных расширений языка.
```

Comment:
I suppose the hyphen is not needed in the fourth sentence but a comma is missing before `как`. Also it probably would be better to merge this sentence with the previous one using a comma between them. So finally it could look like `Мы начнем этот урок с рассмотрения внутреннего представления Elixir кода, а потом узнаем, как его модифицировать`.